resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/features/BasicDriverFeatureStepSuite.scala

```diff
@@ -93,15 +93,16 @@ class BasicDriverFeatureStepSuite extends SparkFunSuite {
       assert(envs(v) === v)
     }
     assert(envs(ENV_SPARK_USER) === Utils.getCurrentUserName())
-
-    assert(configuredPod.pod.getSpec().getImagePullSecrets.asScala ===
-      TEST_IMAGE_PULL_SECRET_OBJECTS)
+    assert(envs(ENV_APPLICATION_ID) === kubernetesConf.appId)
 
     assert(configuredPod.container.getEnv.asScala.exists(envVar =>
       envVar.getName.equals(ENV_DRIVER_BIND_ADDRESS) &&
         envVar.getValueFrom.getFieldRef.getApiVersion.equals("v1") &&
         envVar.getValueFrom.getFieldRef.getFieldPath.equals("status.podIP")))
 
+    assert(configuredPod.pod.getSpec().getImagePullSecrets.asScala ===
+      TEST_IMAGE_PULL_SECRET_OBJECTS)
```

Comment:
Why do you move this to here? I guess your contribution can be minimized to one line (line 96).

Reply:
Oh. I can definitely do that. I notice this line was placed between container env setups. Move this line give all env setup code together. Does it make sense? Or you want to minimize code change?

Reply:
Let's minimize the code change. Usually, it's recommended not to mix the estehtic changes and feature changes.